docs/aop/user/intro-to-aop/aop-overview.md

```diff
@@ -0,0 +1,51 @@
+---
+title: App Orchestration Platform overview
+description: Learn about the App Orchestration Platform and how you can use it.
+template: concept-topic-template
+---
+
+The App Orchestration Platform (AOP) lets [Spryker Cloud Commerce OS (SCCOS)](/docs/cloud/dev/spryker-cloud-commerce-os/getting-started-with-the-spryker-cloud-commerce-os.html) customers connect, configure, and use the available third-party services, or apps, in their application with a click of a button, without development efforts from their side.
+
+![AOP](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/app-orchestration-platform-overview/aop.png)
+
+The following apps are supported:
+
+- PayOne, a Payment Service Provider (PSP)
+- UserCentrics, a Consent Management Platform (CMP)
+- BazaarVoice, a platform for User-Generated Content (UGC)
+
+<!---Add links to the general apps descriptions, once available-->
+
+All integrations of the apps are built by Spryker and provided in a secure and no-code way for the SCCOS.
+
+The main advantages of the AOP are:
```

Comment:
A list should be introduced with a complete sentence, not a partial one that's completed by the list items. Therefore, consider replacing "The main advantages of the AOP are" with something like "The AOP has the following advantages" (see https://developers.google.com/style/lists#introductory-sentences-for-lists)
```suggestion
The AOP has the following advantages:
```